src/modules/commands/discord/russianroulette.ts

```diff
@@ -1,15 +1,24 @@
+import discord from 'discord.js'
 import { Command } from '@commands/CommandHandler'
 import { randomBigInt, randomChoice } from '@app/utils/random'
+import { sleep } from '@app/utils/async'
 
-export default new Command({
+export default new Command<discord.Message>({
   keyword: 'russianroulette',
   description: 'kill yourself for fun purposes',
   help: 'Usage: `{prefix}russianroulette` - kill yourself for fun purposes',
-  callback: ({ args, reply, author }) => {
-    if (args.length > 0) return
+  callback: async ({ author, reply, raw }) => {
```

Comment:
Don't forget to check the number of arguments (return early if > 0).

Reply:
done